lib/workers/pr/changelog/release-notes.ts

```diff
@@ -176,13 +215,21 @@ export async function getReleaseNotesMd(
         `Multiple candidates for changelog file, using ${changelogFile}`
       );
     }
-    const fileRes = await http.getJson<{ content: string }>(
-      `${apiPrefix}/${changelogFile}`
-    );
-    changelogMd =
-      Buffer.from(fileRes.body.content, 'base64').toString() + '\n#\n##';
+    if (apiBaseUrl.includes('gitlab')) {
+      const fileRes = await glGot<{ content: string }>(
+        `${apiFiles}${changelogFile}?ref=master`
+      );
+      changelogMd =
+        Buffer.from(fileRes.body.content, 'base64').toString() + '\n#\n##';
+    } else {
+      const fileRes = await http.getJson<{ content: string }>(
+        `${apiFiles}${changelogFile}`
+      );
+      changelogMd =
+        Buffer.from(fileRes.body.content, 'base64').toString() + '\n#\n##';
+    }
```

Comment:
```suggestion
    let fileRes: { body: { content: string } };
    if (apiBaseUrl.includes('gitlab')) {
      fileRes = await glGot<{ content: string }>(
        `${apiFiles}${changelogFile}?ref=master`
      );
    } else {
      fileRes = await http.getJson<{ content: string }>(
        `${apiFiles}${changelogFile}`
      );
    }

    changelogMd =
      Buffer.from(fileRes.body.content, 'base64').toString() + '\n#\n##';
```
Little simplification